tests/test_ht1621.py

```diff
@@ -107,7 +107,7 @@ def reset_mock(self):
         self._pin_states = {}  # Pin IO states
         self._state = 'none'   # Current state
 
-    def cleanup(self):
+    def cleanup(self, *args):
```

Comment:
this change isn't needed is it?

Reply:
I think it is needed.  The problem is the ``MockHT1621`` class is intended to be a test object for the ``RPi.GPIO`` class and ``RPi.GPIO.cleanup`` optionally accepts an integer or a list of integers.  I am now using that interface to selectively clean up the GPIO pin used by the GPIO and PWM backlight classes within luma.lcd.  See luma.lcd.device.py:93 and 180.